package.json

```diff
@@ -36,7 +36,7 @@
     "node": ">=4.2.4"
   },
   "dependencies": {
-    "mongoose": "^5.1.6"
+    "mongoose": "^5.2.5"
   },
   "devDependencies": {
```

Comment:
Also, please run NPM outdated, and lock all dev dependencies to specific versions.